subsys/net/ip/net_context.c

```diff
@@ -3685,9 +3716,12 @@ int net_context_set_option(struct net_context *context,
 	case NET_OPT_LOCAL_PORT_RANGE:
 		ret = set_context_local_port_range(context, value, len);
 		break;
+	case NET_OPT_IPV4_MCAST_LOOP:
+		ret = set_context_ipv4_mcast_loop(context, value, len);
+    break;
 	case NET_OPT_IPV6_MCAST_LOOP:
-		ret = set_context_ipv6_mcast_loop(context, value, len);
-		break;
+    ret = set_context_ipv6_mcast_loop(context, value, len);
+    break;
```

Comment:
Something went wrong with the indentation. You probably do not need to touch the IPv6 variant here.